src/js/components/Image.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const METHODS_TO_BIND = ['onImageError'];
+
+class Image extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  onImageError(event) {
+    let {fallbackSrc} = this.props;
+    if (!fallbackSrc || event.target.src === fallbackSrc) {
+      return;
```

Comment:
done. Also, had an offline conversation where we tried hiding the container, but that didn't feel right, so we decided not to go with that. See the result here:
![](https://cl.ly/112d20292S3Z/Screen%20Recording%202016-07-15%20at%2013.01.gif)